app/src/main/res/layout-land/ongoing_topic_item.xml

```diff
@@ -57,37 +57,32 @@
 
         <TextView
           android:id="@+id/topic_name_text_view"
+          style="@style/CardTitle1"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:layout_marginStart="8dp"
           android:layout_marginTop="8dp"
           android:layout_marginEnd="8dp"
-          android:ellipsize="end"
-          android:fontFamily="sans-serif"
           android:gravity="start"
           android:maxLines="2"
           android:minLines="2"
           android:text="@{viewModel.topic.name}"
-          android:textAlignment="viewStart"
           android:textColor="@color/white"
-          android:textSize="14sp"
           android:textStyle="bold"
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
           android:id="@+id/story_count_text_view"
+          style="@style/CardTitle2"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
```

Comment:
Remove`layout_height` attribute as it is already mentioned in `Body` style. (Make sure its `wrap_content` only in styles.xml)